functors_task_1/student/include/impl.h

```diff
@@ -16,18 +25,23 @@ struct BeerOrganizer
  *
  * @note Only Corona and HoeGaarden are expensive
  */
-bool isExpensiveBeer(/**???*/)
+bool isExpensiveBeer(const BeerBrand& beer)
 {
+    std::set<BeerBrand> expensiveBeers = {BeerBrand::Corona, BeerBrand::HoeGaarden};
+    return expensiveBeers.find(beer) != expensiveBeers.end();
 }
 
 /**
  * @todo Implement lambda beer country equality comparator
  * @return true if beer county is the same, false otherwise
  */
-auto sameCountry = [](/**???*/)
+auto sameCountry = [](const BeerBrand& first, const BeerBrand& second)
```

Comment:
https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#f16-for-in-parameters-pass-cheaply-copied-types-by-value-and-others-by-reference-to-const